script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt

```diff
@@ -27,37 +27,37 @@ That's why in return―――"
 Arcueid is taken aback by my suicidal remark.
 I am fully aware of how dangerous it is. Even so, there is a line that I cannot cross.
 <Page9>
-"Do you think I can joke at a time like this?
+"You think I could joke about something like this?
 I don't want to die either. So please listen to me. If you rest until tonight, could you defeat Vlov if you had the help of my eyes?"
 <Page10>
 Arcueid is the only one who can judge that.
 That's why... if she decides that it's possible, then I will wager everything on that possibility.
 <Page11>
 "...I see. If you move exactly the way I tell you to, then I guess we'd have the upper hand.
 But is that really okay? It would be much safer to just wait three days you know."
-"...I know that. But this is as much as I can handle. I can't wait any longer." // dropping mentally intentionally here because its awkward in english.
+"...I know that. But this is as much as I can handle. I can't bear to wait any longer." // dropping mentally intentionally here because its awkward in english.
```

Comment:
not sure this is an improvement. It changes the register a bit in a way I don't really think fits with these two.